new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Operator.java

```diff
@@ -3,30 +3,36 @@
 
 package oracle.weblogic.kubernetes.assertions.impl;
 
-import java.util.Random;
+import java.util.HashMap;
 import java.util.concurrent.Callable;
 
-import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
 
-public class Operator {
+public class Operator implements LoggedTest {
```

Comment:
this is not a test -- should not implement LoggedTest
just use a static import to get the logger (this is done in other assertion impls)